addons/models/0001-atlas-authz/0_Base.json

```diff
@@ -0,0 +1,109 @@
+{
+  "entityDefs": [
+    {
+      "name": "Referenceable",
+      "superTypes": [],
+      "serviceType": "atlas_core",
+      "typeVersion": "2.0",
+      "attributeDefs": [
+        {
+          "name": "qualifiedName",
+          "typeName": "string",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": true,
+          "isOptional": false,
+          "isUnique": true,
+          "skipScrubbing": true,
+          "includeInNotification": true,
+          "indexTypeESFields": {
+            "text": {
+              "type": "text",
+              "analyzer": "atlan_text_analyzer"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "Asset",
```

Comment:
How is this Asset different from Asset in models repo ?  And what made us to repeat the model here ?

Reply:
Please check [this comment](https://github.com/atlanhq/atlas-metastore/pull/1736#discussion_r1154315639)

Reply:
Here it will be create with minimal required attributeDefs whereas models repo has all other attributeDefs